uint8 svId                  
uint16 elev                 
uint16 azim                 
uint8 cno                   
float64 prMes
uint8 prStdev